src/precompile_includer.jl

```diff
@@ -14,4 +14,16 @@ elseif !ismultios && !ismultiversion
         _precompile_()
     end
 else
+    @static if v"1.6.0-DEV" <= VERSION <= v"1.6.9" 
```

Comment:
Don't you just have to change this line? Or will it be overwritten once the bot runs again?

Reply:
Yes, it will be overwritten, this file is autogenerated.